portfolio/pom.xml

```diff
@@ -23,6 +23,12 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
```

Comment:
I see you have added GSON but are not using it yet. Was that intentional?

Reply:
I just implemented it in my newest pull request.